static/_redirects

```diff
@@ -584,4 +584,5 @@
 /pt/*         /pt-br/:splat  302!
 /zh/*         /zh-cn/:splat  302!
 
-/docs/concepts/overview/what-is-kubernetes/      /docs/concepts/overview/ 301
\ No newline at end of file
+/docs/concepts/overview/what-is-kubernetes/      /docs/concepts/overview/ 301
+/zh-cn/docs/concepts/overview/what-is-kubernetes/      /zh-cn/docs/concepts/overview/ 301
```

Comment:
I don't think this is necessary although it is nothing wrong to make this change.
My assumption is that no one would bookmark the "what is kubernetes" page.
The case of "breaking user land" simply doesn't exists.